tests/governance.py

```diff
@@ -48,66 +47,6 @@ def test_consensus_status(network, args):
     return network
 
 
-@reqs.description("Test quotes")
-@reqs.supports_methods("/node/quotes/self", "/node/quotes")
-def test_quote(network, args):
```

Comment:
This test is deleted. It's almost exactly the same as `verify_quotes` in `code_update`. The only other thing it does is check that the `/node/quotes/self` calls match entries from the single `/node/quotes` list. That's now added to `verify_quotes`.